SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -301,6 +313,39 @@ void FLocalDeploymentManager::TryStartLocalDeployment(FString LaunchConfig, FStr
 	return;
 }
 
+bool FLocalDeploymentManager::SetupRuntimeFileLogger(FString SpatialLogsSubDirectoryName)
+{
+	// Ensure any old log file is cleaned up
+	if (RuntimeLogFileHandle)
+	{
+		delete RuntimeLogFileHandle;
+		RuntimeLogFileHandle = nullptr;
+	}
+
+	FString RuntimeLogDir = FPaths::Combine(SpatialGDKServicesConstants::LocalDeploymentLogsDir, SpatialLogsSubDirectoryName);
+
+	FString RuntimeLogFilePath = FPaths::Combine(RuntimeLogDir, TEXT("runtime.log"));
+
+	IPlatformFile& PlatformFile = FPlatformFileManager::Get().GetPlatformFile();
+
+	bool bSuccess = PlatformFile.CreateDirectoryTree(*RuntimeLogDir);
+
+	if (bSuccess)
+	{
+		RuntimeLogFileHandle = PlatformFile.OpenWrite(*RuntimeLogFilePath, /*bAppend*/ true, /*bAllowRead*/ true);
+	}
+
+	if (!bSuccess && RuntimeLogFileHandle)
```

Comment:
Should this not be if (bSuccess && RuntimeLogFileHandle == nullptr)

I understand this as catching when  PlatformFile.OpenWrite failed?

Reply:
Ah good catch, yeah I messed this up when refactoring

Reply:
If so maybe we should have this check after: RuntimeLogFileHandle = PlatformFile.OpenWrite(*Runt

Reply:
Should be `(!bSuccess || RuntimeLogFileHandle == nullptr)` actually